app/code/core/Mage/Eav/Model/Entity/Attribute/Frontend/Abstract.php

```diff
@@ -89,6 +89,16 @@ public function getLabel()
         return $label;
     }
 
+    /**
```

Comment:
After, with the last commit I will remove this function because it makes no sense to put it in the "Frontend" class. It is outside the business logic.